Files/ViewModels/Bundles/BundlesViewModel.cs

```diff
@@ -105,6 +105,7 @@ private void InputTextKeyDown(KeyRoutedEventArgs e)
         {
             if (e.Key == VirtualKey.Enter)
             {
+                OnPropertyChanged(nameof(BundleNameTextInput)); // Update the property text
```

Comment:
`UpdateSourceTrigger=PropertyChanged` is enough in XAML, no need to add `OnPropertyChanged`.

Reply:
Thanks! Good to know!